frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalarSettingsWidgets.styled.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from "@emotion/styled";
+import type { MouseEvent } from "react";
+import { Menu, NumberInput } from "metabase/ui";
+
+export const MenuTargetStyled = styled(Menu.Target)`
+  width: 100%;
+
+  .emotion-Button-label {
+    width: 100%;
+    height: 100%;
+  }
+`;
```

Comment:
Can we avoid using a CSS class here directly? If Mantine changes the class name (and it most likely will be different in v7 since they're dropping emotion), the style will break